Sources/PointFree/Account/Account.swift

```diff
@@ -276,7 +276,8 @@ private let profileRowView = View<AccountData> { data -> Node in
 
 private let emailSettingCheckboxes = View<[Database.EmailSetting]> { currentEmailSettings in
   [
-    p(["Receive email when:"]),
+    // TODO: hide `welcomeEmails` for subscribers?
```

Comment:
maybe we card this if we dont wanna address now

Reply:
Should be simple enough. Lemme see.

Reply:
sweet looks like you did it, so we could remove this todo now.